examples/common/python/connectors/direct/tcs_listener/tcs_worker_registry_handler.py

```diff
@@ -312,7 +312,6 @@ def __process_worker_update(self, worker_id, input_json_str, response):
             json_dict = json.loads(value)
             input_value = json.loads(input_json_str)
             worker_details = input_value ["params"]["details"]
-            worker_details = json.loads(worker_details)
```

Comment:
Worker details field would be string and converting it to json object. Worker registry update will update json object and convert it back to string and updating details field at 318. Which test is failing?

Reply:
I believe details param is dictionary not string. that's why json.loads is failing.